redux/reducers/dropdowntoggle.js

```diff
@@ -0,0 +1,21 @@
+import { CLOSE_DROPDOWN, OPEN_DROPDOWN } from '../actionTypes';
+
+const initialState = {
+  dropdownIsOpened: false,
+};
+
+export default function dropdownToggle(state = initialState, action) {
```

Comment:
Can you make a custom hook for this logic? So that it will be reusable for other modal components as well. Here is the reference https://usehooks.com/useOnClickOutside/

Reply:
Sure I will👍🏻

Reply:
https://usehooks.com/useOnClickOutside/